sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CAutoConfiguration.java

```diff
@@ -124,13 +124,15 @@ private void addB2CClientRegistration(@NonNull List<ClientRegistration> registra
         @Bean
         @ConditionalOnMissingBean
         public ClientRegistrationRepository clientRegistrationRepository() {
-            final List<ClientRegistration> registrations = new ArrayList<>();
+            final List<ClientRegistration> signUpOrSignInRegistrations = new ArrayList<>();
```

Comment:
Will we have several signup or signin flows?

Reply:
for now, our configuration file can only config one flow of signUpOrSignIn.